### DISCUSSION TITLE
formatting a simple formula output makes result unusable for other formula

### DISCUSSION THREAD

**Discussion by dominiklenne** [freeplane/freeplane#288] [General]
dear forists,
I happily discovered how to use simple arithmetic formulas in FP using the "=" sign.
Now I'd like to limit the number of decimal places in the output.

  =format(ID_1822300691/ID_56542080,'0.0')

does the trick, but yields a number with a comma as decimal separator: 

81,3

I understand that this is the format command, that puts the DE localization on the number.
When I copypaste this, I see:

81.30081300813008

which is the correct internal representation.

Now when I do any other operation with the result, i.e.  / 0.9, I get:

0.0

which is obviously wrong. It seems as if the program does not take the internal representation of the number as input, but the string '81,3' which is evaluated as 0. 

Any constructive thoughts on this? What am I making wrong?  - - - many thanks in advance...

**Comment by Illegit** [comment]
I use format: 
=format(value,'#0.00') for having 2 decimals of "value"
SFPC

**Comment by EdoFro** [comment]
I don't use _format()_ in formulas.

It is much easier to go to the **format tab** in the **tool panel** and pick a **number format**  in the **Core text** section.

You can assign that to a Style that you use for nodes with formulas.

![image](https://user-images.githubusercontent.com/35700575/161340533-f57e05e5-aedc-4645-83bd-4dde83e64976.png)

one node has this text:
`=100/9`

and its child:
`= node.parent / 2`


By doing it in this way you don't lose precision when using the result in other formulas.

Hope it helps

**Reply by dominiklenne** [reply]
a good tip. thx i'll try it.

**Reply by dominiklenne** [reply]
works brillantly. Could you give a hint how to make a style out of this? thx.
